components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/config/WebSubAdapterConfiguration.java

```diff
@@ -73,4 +95,44 @@ public String getWebSubHubBaseUrl() {
 
         return webSubHubBaseUrl;
     }
+
+    /**
+     * Returns the HTTP connection timeout.
+     *
+     * @return HTTP connection timeout.
+     */
+    public Integer getHTTPConnectionTimeout() {
```

Comment:
These won't be null, any reason why we use `Integer` instead of primitive `int`?